src/reducers/url-state.js

```diff
@@ -292,7 +304,7 @@ export const getTransformStack = (
   state: State,
   threadIndex: ThreadIndex
 ): TransformStack => {
-  return getURLState(state).transforms[threadIndex] || [];
+  return getURLState(state).transforms[threadIndex] || EMPTY_TRANSFORM_STACK;
```

Comment:
definitely useful :) 👍

Reply:
I only kept this one. My initial thought was to mechanically replace everything so it would be easy to catch this type of error. I don't think the other one you suggested I keep would have helped in memoization unless I'm missing something, so I removed it as well. I don't know what you mean by "accessor" though.